js/actionCreators/files.ts

```diff
@@ -198,6 +198,29 @@ export function fetchMediaDuration(url: string, id: number): Dispatchable {
   };
 }
 
+export function loadMedia(
+  e: React.DragEvent<HTMLDivElement>,
+  loadStyle: LoadStyle = LOAD_STYLE.NONE,
+  atIndex = 0
+): Dispatchable {
+  return (dispatch, getState, { handleTrackDropEvent }) => {
+    const defaultHandler = loadFilesFromReferences(
+      e.dataTransfer.files,
+      loadStyle,
+      atIndex
+    );
+
+    if (handleTrackDropEvent)
+      handleTrackDropEvent(e).then(tracks => {
+        if (tracks) dispatch(loadMediaFiles(tracks, loadStyle, atIndex));
+        else {
+          dispatch(defaultHandler);
+        }
+      });
+    else dispatch(defaultHandler);
+  };
```

Comment:
You can run `npm run format` from the root directory to apply the prettier formatting, which will keep the code formatting in line with the rest of the project.